requirements/constraints.txt

```diff
@@ -8,7 +8,12 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-# These packages are backports which can only be installed on Python 2.7
-futures ; python_version == "2.7"
 
 
+# Stay on an LTS release
+django<2.3              # Web application framework
+
+
+
+# pip20+ requires for greater version of pip-tools. But installing 19.3.1 in all our containers and VMs.
+pip-tools==4.5.1
```

Comment:
I've applied a change to the `make upgrade` Jenkins jobs that should let us upgrade pip-tools in library packages like this one: https://github.com/edx/jenkins-job-dsl/pull/953 .  We probably want to keep it pinned in IDAs until we switch the configuration repo to build Docker images with newer pip versions, though.